migrations/20220426234444_support-suspending-users.js

```diff
@@ -0,0 +1,23 @@
+// Ref: https://github.com/voxpelli/node-connect-pg-simple/blob/main/table.sql
+
+exports.up = function up(knex) {
+  return knex.schema.raw(`
+    alter table public.user add column is_suspended boolean not null default false;
+
+    create table user_session (
+        sid text primary key
+      , sess json not null
```

Comment:
Nitpick: SQL lines end with a comma, and not start with one, it's also the style we've been following so far